README.md

```diff
@@ -56,14 +56,25 @@ Each user is allowed to have different local specifications: We don't put any re
 
-There's no "default" for `user-config.yaml` because it depends on system specifications and user preferences. Two things we do recommend keeping in `user-config.yaml` are the name of your Stata executable—which we'll prompt you to enter if necessary—and the location of a [SCons cache directory](http://scons.org/doc/2.0.1/HTML/scons-user/c4213.html)—if you have one. A Mac example where Example_User is running a factory-fresh StataMP and has local access to a directory named cache/template on Dropbox would be 
+There's no "default" for `user-config.yaml` because it depends on system specifications and user preferences. Three things we do recommend keeping in `user-config.yaml` are the name of your Stata executable, the location of a [SCons cache directory](http://scons.org/doc/2.0.1/HTML/scons-user/c4213.html), and the location of a release directory. These fields don't have to be specified if you're not using them, and we'll prompt you for their values at runtime if you've forgotten to specify them and they're necessary. A Mac example where Example_User is running a factory-fresh StataMP and has local access to a directories named cache/template and release on Dropbox would be 
 
 ```YAML
-stata_executable: statamp
+stata_executable: StataMP
 cache_directory: /Users/Example_User/Dropbox/cache/template
 release_directory: /Users/Example_User/Dropbox/release/
 ```
 
+##### How do I handle data external to my repository?
+
+We are agnostic about how you incorporate external data into the template. There's no custom builder for these assets, by design. We expect data storage and transfer protocols to differ—with good reason—across projects and labs, and we try to keep the template flexible enough for any use. That said, we do have some suggestions:
+
+* When a large dataset is stored locally, `user-config.yaml` can include an entry specifying the user-specific path to that dataset. The key of the entry should be constant across users and documented in the top-level readme of the repository. Scripts can access the path to the dataset by parsing `user-config.yaml` and referencing the key. 
+
+* When a large dataset is stored externally, there are a few options. 
+    * The top-level readme can specify manual download and storage instructions. This is simple, easy to customize, and unlikely to cause errors during a SCons build. It does, however, require each user to successfully download the same dataset, perhaps in an unstructured manner. 
+    * The download can be incorporated into the SCons build by either specifying the download steps directly as standard SCons commands or including the download steps in scripts executed by our custom builders. These methods have the benefits of automation and dependency tracking, but they can introduce idiosyncratic errors if the download steps are prone to fail. This can be irksome—since the build may often fail—or dangerous—as repeated builds with identical code may produce different output. 
```

Comment:
I'd replace this with more concrete examples of what we do. I.e. say we either use a command to call a downloader (e.g. `rclone`) or include the download step in a Python script. 

I'd also cut
"This can be irksome—since the build may often fail—or dangerous—as repeated builds with identical code may produce different output. "

Anyone using `gslab_scons` to get an automated build already knows this.